client/components/split-button/index.jsx

```diff
@@ -61,12 +59,12 @@ class SplitButton extends Component {
 
 	hideMenu = () => this.toggleMenu( false );
 
-	toggleMenu( isMenuVisible ) {
+	toggleMenu = isMenuVisible => {
```

Comment:
This is only invoked as `this.toggleMenu` and not passed outside of this scope. It doesn't look like it needs to become an arrow method.

Reply:
thanks @sirreal! I'm going to leave this for now as a style choice. there was already a discussion about it in another PR where I didn't change it to an arrow function.

basically: I think it's a good direction to make all method functions arrow functions unless we have a compelling reason not to (lexical scoping is less surprising than dynamic scoping and easier to debug) while others feel we need a warrant to shift (people should know whether a function they call depends on dynamic scoping and we save on one function allocation per component instance).